### PR TITLE
ANDROAPP-6535-Bidirectional-relationships-are-created-in-the-oppsite-direction-when-created-from-TO

### PR DESCRIPTION
**Pull request by andresmr**
## Description
Fix create bidirectional relationship creation

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-6535)

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ kotlin = '2.0.20'
 hilt = '2.47'
 jacoco = '0.8.10'
 designSystem = "0.4.1-SNAPSHOT"
-dhis2sdk = "1.11.1-SNAPSHOT"
+dhis2sdk = "1.11.1-20241218.080719-10"
```
Comment (Balcan) [inline gradle/libs.versions.toml:11]: Should we need to wait until this is in the snapshot version?

**Review by Balcan — APPROVED** [review]
Very nice!

File: gradle/libs.versions.toml (hunk 1 above)
Comment (andresmr) [reply gradle/libs.versions.toml:11]: yes we are going to wait before merge

**Review by xavimolloy — COMMENTED** [review]
Amazing job!

**Review by xavimolloy — APPROVED** [review]

**Pull request merged**